packages/nuxi/src/commands/build.ts

```diff
@@ -52,6 +52,9 @@ export default defineNuxtCommand({
     await buildNuxt(nuxt)
 
     if (args.prerender) {
+      if (!nuxt.options.ssr) {
+        consola.warn('HTML content not prerendered because `ssr: false` was set.')
```

Comment:
Small note: This log is for the future (the content "that will be prerendered")

Reply:
This is printed out after the content is already prerendered.